services/autoscaling/src/simcore_service_autoscaling/modules/auto_scaling_core.py

```diff
@@ -177,6 +185,29 @@ def _sort_according_to_allowed_types(instance_type: EC2InstanceType) -> int:
     return allowed_instance_types
 
 
+async def _activate_and_notify(
+    app: FastAPI,
+    auto_scaling_mode: BaseAutoscaling,
+    drained_node: AssociatedInstance,
+    tasks: list,
+) -> list:
+    await asyncio.gather(
+        *(
```

Comment:
why do you unpack here? i.e. why to use the  `*` operator in here? aren't these three tasks? i.e. 
```python
await asyncio.gather( set_node(...),  log_message(...),  progress_message(...) )
```

Reply:
good point